QUICKSTART-LOCAL.md

```diff
@@ -0,0 +1,169 @@
+For running a local VM, neither mechanism can be used to apply configuration on first boot: Bottlerocket is not yet running, making its API server unavailable, and the goal to have Bottlerocket running locally precludes use of the user data mechanism.
+As an alternative, the `start-local-vm` wrapper script included in the `tools` directory of the main repository allows to inject configuration into well-known locations of the built image for Bottlerocket to find on boot.
+
+
+### Set up networking
+
+The `start-local-vm` wrapper configures QEMU to provide one virtual network interface to the VM.
+To enable this interface, create a file named `net.toml` containing the following TOML snippet:
+
+```
+version = 1
+
+[enp0s16]
+dhcp4 = true
+```
+
+This will prompt [netdog](https://github.com/bottlerocket-os/bottlerocket/blob/develop/sources/api/netdog/README.md) to set up `enp0s16` as the primary network interface with IPv4 networking configured via DHCP.
+No dedicated DHCP server needs to be running on the host as QEMU will act as one on the virtual network interface.
+Note that for virtual machines launched with `start-local-vm`, the primary network interface will always be named `enp0s16`.
+The name will differ when running on bare metal or in a cloud environment.
+
+
+### Accessing your Bottlerocket guest via host containers
+
+When running a Bottlerocket development variant such as metal-dev locally, you can directly interact with the system via the serial console that `start-local-vm` connects you to.
+For remote access to your running Bottlerocket VMs, you will need to provide additional configuration to enable host containers.
+The Bottlerocket metal images don't include any host containers enabled by default.
+But don't worry!
+You can use our [admin](https://github.com/bottlerocket-os/bottlerocket-admin-container) and/or [control](https://github.com/bottlerocket-os/bottlerocket-control-container) containers, they just need to be configured first.
+Information about the roles these host containers play can be found [here](https://github.com/bottlerocket-os/bottlerocket/#exploration).
+
+
+#### Admin container
+
+If you would like to use the admin container, you will need to create some base64 encoded user data which will be passed to the container at runtime.
+Full details are covered in the [admin container documentation](https://github.com/bottlerocket-os/bottlerocket-admin-container#authenticating-with-the-admin-container).
+If we assume you have a public key at `${HOME}/.ssh/id_rsa.pub`, the below will add the correct user data to your `user-data.toml`.
+
+```
+PUBKEY_FILE="${HOME}/.ssh/id_rsa.pub"
+PUBKEY=$(< "${PUBKEY_FILE}")
+ADMIN_USER_DATA="$(echo '{"ssh": {"authorized-keys": ["'"${PUBKEY}"'"]}}' | base64 -w 0)"
+
+cat <<EOF >>user-data.toml
+[settings.host-containers.admin]
+enabled = true
+user-data = "${ADMIN_USER_DATA}"
+source = "public.ecr.aws/bottlerocket/bottlerocket-admin:v0.9.0"
```

Comment:
This is a bit awkward because it needs to be kept up to date. However, it seems the defaults defined in `sources/models/shared-defaults/public-host-containers.toml` are replaced entirely with the `user-data.toml` placed directly on the file system, i.e. if the `source` is left off here, thar-be-settings will notice and error out on boot.

1. Any ideas for forcing a merge?
2. The public admin and control containers don't have a `latest` tag which would be helpful here. Are there downsides to introducing one and referring to it instead?

Reply:
Of course we would like to keep this reasonably up to date, but for the purposes of this guide using that container version is not going to break at any point in the future.

The wider ramifications of creating a `latest` tag are probably not worth it for the purpose of setting up the host containers in a quickstart.